applications/ChimeraApplication/python_scripts/navier_stokes_ale_chimera_fluid_solver.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+# Import base class file
+from KratosMultiphysics.FluidDynamicsApplication.navier_stokes_ale_fluid_solver import NavierStokesAleFluidSolver
+import KratosMultiphysics.ChimeraApplication.python_solvers_wrapper_fluid_chimera as solver_factory
+
+def CreateSolver(model, custom_settings):
+    return AleChimeraFluidSolver(model, custom_settings)
+
+class AleChimeraFluidSolver(NavierStokesAleFluidSolver):
+    "this class is used only when used defined movement of patch is required"
+    def __init__(self,model,project_parameters, parallelism="OpenMP"):
```

Comment:
```suggestion
    def __init__(self,model,project_parameters, parallelism):
```

better to make this explicit